flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+"only-arches": [
+  "x86_64"
```

Comment:
this is disappointing.

Reply:
Currently the application is build for x86 only